docs/Migration-from-1.6-to-1.7.md

```diff
@@ -198,4 +198,19 @@ If you still have to compile our SDK for older operating systems, then you need
    ./scripts/ios-build-extensions.sh extensions watchos --legacy-archs --use-bitcode --out-dir ./Build
    ```
 
-If you use cocoapds for PowerAuth mobile SDK integration, then please let us know and we'll prepare a special release branch for you.
\ No newline at end of file
+If you use cocoapds for PowerAuth mobile SDK integration, then please let us know and we'll prepare a special release branch for you.
+
+## Changes in 1.7.5+
+
+### iOS
+
+The behavior of `PowerAuthSDK.authenticateUsingBiometry()` has been slightly changed and improved:
+
+- Function now properly treat biometry lockout and increase failed attempts counter on the server, See [Biometry lockout](PowerAuth-SDK-for-iOS.md#biometry-lockout) chapter.
+- Function now returns new `PowerAuthErrorCode_BiometryFallback` error code in case that user tap on the fallback button.
+- As a benefit, function now properly handles situations when the user press home or power button during the biometric authentication.
+- You can also cancel the pending authentication via the application provided `LAContext` object.
```

Comment:
Application provided? This is weird to me, I would use "platform" or "system" provided

Reply:
Application must provide own `LAContext` instance as a parameter to the call and then it can use `invalidate` method to cancel the pending authentication.

Reply:
I'm using this wording because SDK itself create own LAContext internally, when developer use variant with prompt string at input.